lab-aaron/lib/read-file.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+let yoda = module.exports = {};
+
+const fs = require('fs');
+
+yoda.read = (callback) => {
+  let first, second, third;
+
+  fs.readFile(`${__dirname}/../assets/one.txt`, (err, data) => {
```

Comment:
these were all supposed to have console.log messages that prints the first 8 bytes to the terminal